app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,261 @@
+<!--
+  ~ Copyright (c) 2019. Fulton Browne
+  ~  This program is free software: you can redistribute it and/or modify
+  ~     it under the terms of the GNU General Public License as published by
+  ~     the Free Software Foundation, either version 3 of the License, or
+  ~     (at your option) any later version.
+  ~
+  ~     This program is distributed in the hope that it will be useful,
+  ~     but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~     GNU General Public License for more details.
+  ~
+  ~     You should have received a copy of the GNU General Public License
+  ~     along with this program.  If not, see <https://www.gnu.org/licenses/>.
+  -->
+
+<resources>
+    <string name="app_name">Ara</string>
+    <string name="action_settings">die Einstellungen</string>
+    <string name="action_about">Über</string>
+    <string name="greet">Hallo, kann ich helfen?</string>
+    <string name="greet1">Hallo, @string/greet</string>
+    <string name="blank" />
+    <string name="title_activity_settings">die Einstellungen</string>
+
+    <!-- Strings related to Settings -->
+
+    <!-- Example General settings -->
+    <string name="pref_header_general">Allgemeines</string>
+
+    <string name="pref_title_social_recommendations">Aktivieren Sie soziale Empfehlungen</string>
+    <string name="pref_description_social_recommendations">
+	
+Empfehlungen für Kontaktpersonen basierend auf Ihrem Nachrichtenverlauf
+    </string>
+  <string name="botton_menu_title">Zuhause</string>
+  <string name="new_app_widget2_title">WIDGET KOMMT BALD</string>
+
+    <string name="pref_title_display_name">Anzeigename</string>
+    <string name="pref_default_display_name">John Smith</string>
```

Comment:
The german equivalent is "Max Mustermann"

Reply:
for the names or for `line #37`?

Reply:
for the default name - not sure if changing is necessary